src/services/monitoring.ts

```diff
@@ -10,422 +22,337 @@ const {
-          specialists: [
-            {
-              name: 'Specialist 1',
-              roles: ['GS'],
-            },
-            {
-              name: 'Specialist 1',
-              roles: ['ECS'],
-            },
-          ],
-          objectives: [
-            {
-              title: 'The TTA Specialist will support the recipient by developing a plan to ensure all staff implement health and safety practices to always keep children safe',
-              activityReportIds: ['14AR12345'],
-              endDate: '06/24/2024',
-              topics: ['Human Resources', 'Ongoing Monitoring Management System', 'Safety Practices', 'Training and Profressional Development'],
-              status: 'Complete',
-            },
-          ],
-        },
-      ],
-    },
-    {
-      citationNumber: '1302.47(b)(5)(iv)',
-      status: 'Active',
-      findingType: 'Deficiency',
-      category: 'Inappropriate Release',
-      grantNumbers: ['14CH123456'],
-      lastTTADate: '07/25/2024',
-      reviews: [
-        {
-          name: '241234RAN',
-          reviewType: 'RAN',
-          reviewReceived: '06/21/2024',
-          outcome: 'Deficient',
-          findingStatus: 'Active',
-          specialists: [
-            {
-              name: 'Specialist 1',
-              roles: ['GS'],
-            },
-          ],
-          objectives: [
-            {
-              title: 'The TTA Specialist will assist the recipient with developing a QIP and/or corrective action plan to address the finding with correction strategies, timeframes, and evidence of the correction.',
-              activityReportIds: ['14AR29888'],
-              endDate: '07/12/2024',
-              topics: ['Communication', 'Quality Improvement Plan/QIP', 'Safety Practices', 'Transportation'],
-              status: 'In Progress',
-            },
-          ],
-        },
-      ],
-    },
-    {
-      citationNumber: '1302.91(a)',
-      status: 'Corrected',
-      findingType: 'Noncompliance',
-      category: 'Program Management and Quality Improvement',
-      grantNumbers: ['14CH123456', '14HP141234'],
-      lastTTADate: '06/24/2024',
-      reviews: [
-        {
-          name: '241234FU',
-          reviewType: 'Follow-up',
-          reviewReceived: '10/17/2024',
-          outcome: 'Compliant',
-          findingStatus: 'Corrected',
+  const grantNumbers = await grantNumbersByRecipientAndRegion(recipientId, regionId) as string[];
+
+  const citations = await MonitoringStandard.findAll({
+    include: [
+      {
+        model: MonitoringStandardLink,
+        as: 'standardLink',
+        required: true,
+        include: [
+          {
+            model: MonitoringFindingStandard,
+            as: 'monitoringFindingStandards',
+            required: true,
+            include: [
+              {
+                model: MonitoringFindingLink,
+                as: 'findingLink',
+                required: true,
+                include: [
+                  {
```

Comment:
Also join in MonitoringFidningGrantee to the finding links
then in a top level where join MonitoringRevviewGrantee.granteeId = MonitoringFidningGrantee.granteeId

Reply:
Does this join need to be added to the reviews query as well, @GarrettEHill ?